### PR TITLE
Mzmine 3.9

### PR DESCRIPTION
**Pull request by bernt-matthias**
Also fixes https://github.com/galaxyproject/tools-iuc/issues/5542

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bernt-matthias** [comment]
Test don't work unless we have planemo with 23.1 packages...

Tested it locally

File: tools/mzmine/mzmine_batch.xml [hunk 1]
```diff
@@ -30,9 +31,16 @@
     ## modify output paths in batch XML to relative path
     ## - any output will be redirected to ./output/BASENAME
     mkdir -p output/ &&
-    
-    sed -e 's@<current_file>.*[\\\/]\([^\\\/]\+\)</current_file>@<current_file>./output/\1</current_file>@' '$batch' > batch.xml &&
+    python '$__tool_directory__'/set_path.py
```
Comment (bgruening) [inline tools/mzmine/mzmine_batch.xml:34]:
```suggestion
    python '$__tool_directory__/set_path.py'
```

File: tools/mzmine/mzmine_batch.xml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-        <token name="@TOOL_VERSION@">3.6.0</token>
+        <token name="@TOOL_VERSION@">3.9.0</token>
         <token name="@VERSION_SUFFIX@">0</token>
     </macros>
     <xrefs>
         <xref type="bio.tools">mzmine</xref>
     </xrefs>
     <requirements>
         <requirement type="package" version="@TOOL_VERSION@">mzmine</requirement>
+        <requirement type="package" version="3.11">python</requirement>
```
Comment (bgruening) [inline tools/mzmine/mzmine_batch.xml:11]: Why is this needed?
```suggestion
```

File: tools/mzmine/mzmine_batch.xml [hunk 3]
```diff
@@ -54,20 +62,49 @@
         <param argument="--batch" type="data" format="xml" label="MZMine batch file" help="XML batch file (ideally created with version @TOOL_VERSION@)"/>
         <param argument="--input" type="data" format="mzml,mzxml,csv" multiple="true" label="Spectra (mzml)"/>
         <param argument="--libraries" type="data" format="json,mgf,msp" multiple="true" optional="true" label="Spectral libraries"/>
+        <param name="localdb" type="data" format="csv" optional="true" label="Local DB (CSV)" help="Only used of the batch XML file contains a LocalCSVDatabaseSearchModule step"/>
```
Comment (bgruening) [inline tools/mzmine/mzmine_batch.xml:65]: The help section is hard to understand.

File: tools/mzmine/mzmine_batch.xml (hunk 2 above)
Comment (bernt-matthias) [reply tools/mzmine/mzmine_batch.xml:11]: Needed for the helper script `set_path.py`.

**Review by bgruening — APPROVED** [review]
